demos/serverless/README.md

```diff
@@ -29,3 +29,8 @@ node ./deploy.js -r us-east-1 -b <my-bucket> -s <my-stack-name> -a meeting
 The script will create an S3 bucket and CloudFormation stack
 with Lambda and API Gateway resources required to run the demo. After the script
 finishes, it will output a URL that can be opened in a browser.
+
+
+### Cleaning up
+To avoid incurring future charges, please delete any resources in your account that you are not using. 
```

Comment:
Suggest: To avoid incurring any unintended charges as a result of deploying the serverless demo, it is important to delete the AWS CloudFormation stack after you are finished using it.

Reply:
Updated.